cmd/juju/application/scaleapplication.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package application
+
+import (
+	"strconv"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"gopkg.in/juju/names.v2"
+
+	"github.com/juju/juju/api/application"
+	"github.com/juju/juju/apiserver/params"
+	"github.com/juju/juju/cmd/modelcmd"
+)
+
+// NewScaleApplicationCommand returns a command which scales an application's units.
+func NewScaleApplicationCommand() modelcmd.ModelCommand {
+	cmd := &scaleApplicationCommand{}
+	cmd.newAPIFunc = func() (scaleApplicationAPI, error) {
+		root, err := cmd.NewAPIRoot()
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+		return application.NewClient(root), nil
+	}
+	return modelcmd.Wrap(cmd)
+}
+
+// scaleApplicationCommand is responsible for destroying application units.
+type scaleApplicationCommand struct {
+	modelcmd.ModelCommandBase
+	modelcmd.CAASOnlyCommand
+
+	newAPIFunc      func() (scaleApplicationAPI, error)
+	applicationName string
+	scale           int
+}
+
+const scaleApplicationDoc = `
+Scale a Kubernetes application by specifying how many units there should be.
```

Comment:
Worth mentioning that it cater for both scale up and scale down? Originally was reading it as a scale up only... until I got bi-directional brilliance...